app/presenters/publishing_page_presenter.rb

```diff
@@ -73,7 +73,7 @@ def submission_warning_presenters
 
     return presenters if deployment_environment == 'dev'
 
-    presenters.push(from_address_warning) if service_output_warning.blank?
+    presenters.push(from_address_warning) if service_output_warning.blank? && ENV['REPLY_TO'] == 'disabled'
```

Comment:
Might be worth doing something like !ENV['REPLY_TO'] == 'enabled' to cover the empty string case, but it's not a huge deal